### DISCUSSION TITLE
Translations?

### DISCUSSION THREAD

**Discussion by pascalstockert** [pascalstockert/yolol#68] [Ideas]
Softwerker recently brought up that translations into other languages could be nice - and I agree! Prismic offers support for translation on pages by default which are already set up, however the site itself would need to be adapted. I haven't looked into how different the API requests will have to be and what kind of stuff we get from them in return. Looking at it from a worst-case perspective this feature would require quite the chunk of time.
Adding to that is the content that would still be missing after the feature itself is implemented - adding another huge chunk of time to the already limited amount of resources we have.

What do you think?

**Comment by pascalstockert** [comment]
TL;DR:
If we were to add translations it should happen ASAP as to not create an overly large overhead in non-translated stuff, ultimately delaying development on other features, bugfixes and chapters.

**Reply by Azurethi** [reply]
Think it's definitely worth it, expecially if people are offering to do the translating. Plus, afaik their api tends to be dev-friendly

**Reply by pascalstockert** [reply]
Aye, I'll have to look into it a bit more but as far as I can tell it should be manageable. The other thing would be how to include voluntary translators in the translation workflow. I already have the English version written down as .md in the repo - maybe just have people open PRs with their translated .mds?
